js/views/message.js

```diff
@@ -167,8 +231,6 @@ views.Messages = Backbone.Marionette.CompositeView.extend({
 	currentMessageId: null,
 
 	events: {
-		"click #load-new-mail-messages" : "loadNew",
-		"click #load-more-mail-messages" : "loadMore"
 	},
```

Comment:
I'm confused. This was deleted a while back and isn't showing up for me. Which commit ID is this?